src/main.rs

```diff
@@ -1,21 +1,62 @@
+//! Default Compute@Edge template program.
+
+use fastly::http::response::Builder as ResponseBuilder;
 use fastly::http::{HeaderValue, Method, StatusCode};
 use fastly::request::CacheOverride;
 use fastly::{downstream_request, Body, Error, Request, RequestExt, Response, ResponseExt};
 use std::convert::TryFrom;
 
-const VALID_METHODS: [Method; 3] = [Method::HEAD, Method::GET, Method::POST];
+/// The name of a backend server associated with this service.
+///
+/// This should be changed to match the name of your own backend.
+const BACKEND_NAME: &str = "backend_name";
```

Comment:
I would really like a link to some sort of `docs.fastly` page somewhere that explains how/where to configure backends in our UI or via the CLI, but I expect C@E is young enough that this doesn't exist yet. Lmk if you have any ideas!

As mentioned in the description, I want to highlight this because it **must** be changed.